src/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './components'
 export { default as theme } from './theme'
 export type { SCWUITheme } from './theme'
 export * from './utils'
+export { createTheme } from './theme'
```

Comment:
This is an extend theme rather than a create no ?

Reply:
@chambo-e Kinda yeah, I used createTheme as it create theme from what is given and it seems like MUI are doing same but I'm open if you want to rename it